codegen/src/generator/rest_json.rs

```diff
@@ -115,6 +115,78 @@ impl GenerateProtocol for RestJsonGenerator {
     }
 }
 
+fn http_code_to_status_code(code: Option<i32>) -> String {
+    match code {
+        Some(actual_code) => {
+            // From http://hyper.rs/hyper/v0.9.12/src/hyper/src/status.rs.html and modified to return strings:
```

Comment:
This bit of code is vendored because I couldn't find a way to turn an instance of an enum into a string representation of itself.  The data doesn't appear to be there and I couldn't find anything enlightening on the internet.

Not sure how I feel about this.  Any rename of a StatusCode enum entry in hyper/reqwest will cause this to break.